docs/proposals/logging_rearch.md

```diff
@@ -0,0 +1,55 @@
+# Re architect origin-aggregated-logging to use updated API objects and Templates to install
+
+## Why?
+
+Currently when we develop features or make changes that require configuration
+updates as well as updates to our images we must coordinate those changes in two
+repositories which can lead to things being out of sync or missed.
```

Comment:
2 repositories is a big problem, but how will the usage of templates fix that?

Reply:
Because we will keep the templates in origin-aggregated-logging?  Then all of the configuration/images will be in one place.

Reply:
Correct, then the role itself will just `oc process -f | oc apply -f -` the templates we provide. We'd keep the majority of the logic in this repo with our images.

Then we can get a Jenkins job set up to automatically sync our template install files over to openshift-ansible for us.

Reply:
So configmaps will fully be in the templates now, not like it is done here? https://github.com/openshift/openshift-ansible/blob/master/files/origin-components/apiserver-template.yaml#L79

Reply:
Correct

Reply:
I need to update this, after having a discussion with Fabian to try to address concerns that Peter had, we'll likely move to an Ansible Playbook Bundle approach. We can choose to use OCP templates or continue to use roles. Whichever we are more comfortable with. I think the intent would be to still use templates as we should leverage `oc` being idempotent.